dotCMS/src/main/java/com/dotmarketing/business/web/HostWebAPI.java

```diff
@@ -31,6 +31,8 @@
 	
 	public Host getCurrentHost(HttpServletRequest req) throws DotDataException, DotSecurityException, PortalException, SystemException;
 
+	public Host getCurrentHost() throws DotDataException, DotSecurityException;

```

Comment:
![Codacy](https://app.codacy.com/assets/images/favicon.png) Issue found: [Unnecessary modifier 'public' on method 'getCurrentHost': the method is declared in an interface type](https://app.codacy.com/manual/dotCMS/core/pullRequest?prid=5067852)

Reply:
Doc

Reply:
done https://github.com/dotCMS/core/pull/18033/commits/286b9c09f52d6e0bb78ec9761ec3000c06d7f38a#diff-875569de59a68af429a7ac46a979b1d9R40